applications/luci-app-smartdns/htdocs/luci-static/resources/view/smartdns/smartdns.js

```diff
@@ -639,21 +817,21 @@ return view.extend({
 		s.tab('advanced', _('Advanced Settings'));
 
 		// enable flag;
-		o = s.taboption("general", form.Flag, "enabled", _("Enable"));
+		o = s.taboption("general", form.Flag, "enabled", _("Enable"), _("Enable"));
```

Comment:
please keep these stuffs as-is, see:
https://github.com/immortalwrt/luci/commit/16f4011644b8689fffb0866e2de4e821b2f52d9b

Reply:
I tried to understand this change, but it was not appropriate. Here is my opinion, this was to provide explanatory text, from the original author's intention.
Here is original code:
![image](https://github.com/immortalwrt/luci/assets/31793080/e2e36794-ed9a-4099-970e-f3a5dd8d9073)
Here is modified code：
![image](https://github.com/immortalwrt/luci/assets/31793080/f3607ee1-84a8-4d09-a000-2f7d63abd530)
And all the javascript has some same behavior.
Eventually, keep same from upstream is better choice.

Reply:
> Here is my opinion, this was to provide explanatory text

IMO this does not really "explain" anything, it's just duplicating the title more or less.

This might be more clear in Chinese: https://github.com/immortalwrt/luci/pull/387